src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/CorporateBilledCorporatePaidSourceAccountingLine.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+	
+	<bean id="CorporateBilledCorporatePaidSourceAccountingLine" parent="CorporateBilledCorporatePaidSourceAccountingLine-parentBean" />
+	<bean id="CorporateBilledCorporatePaidSourceAccountingLine-parentBean" abstract="true" parent="ProcurementCardSourceAccountingLine-parentBean">
```

Comment:
There are several new "parentBean" beans in this PR whose parents are another "parentBean" instead of the actual bean. Should these be referencing the non-"parentBean" version instead?